### PR TITLE
Feat : Added Profile Page UI

### PR DESCRIPTION
**Pull request by srejitk**
The following feature request adds the following;

1.Profile Page UI

### CODE REVIEW

File: social-circus/src/routes/Profile/Profile.jsx [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
+          </h1>
+          <h1 className="mx-auto text-lg font-medium text-gray-400 ">
+            @zenitsu
+          </h1>
+          <p className="mx-auto w-full px-8 pt-4">
+            my bio. That's my existence .
+          </p>
+
+          <p className="mx-auto w-full px-8 py-1 font-semibold text-blue-500">
+            thunderclap.com
```
Comment (aishwarya-bm) [inline social-circus/src/routes/Profile/Profile.jsx:30]: Anchor tag would be better for quick navigation.

**Review by aishwarya-bm — APPROVED** [review]
Great usage of css utils from the library, good to see that there is no custom CSS required.

**Review by Hamdrive — APPROVED** [review]
- Good work implementing the basic structure for profile page
- Overall looks good
- Keep on building!

File: social-circus/src/routes/Profile/Profile.jsx (hunk 1 above)
Comment (srejitk) [reply social-circus/src/routes/Profile/Profile.jsx:30]: Yes, will correct that.

**Pull request merged**